src/main/java/org/b612foundation/adam/datamodel/PropagationParameters.java

```diff
@@ -0,0 +1,82 @@
+package org.b612foundation.adam.datamodel;
+
+import java.util.Objects;
+
+public class PropagationParameters {
```

Comment:
PropagationRequest?

Reply:
I called it PropagationParameters because I think it's reasonable for someone to upload an ephemeris and want to attach propagation parameters describing its origin, even if ADAM didn't propagate it.

Reply:
Mmmm... I am not enthusiastically convinced, but let it be as is.